src/react-app-env.d.ts

```diff
@@ -1 +1,20 @@
 /// <reference types="react-scripts" />
+
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+}
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user: User,
+}
+
+export interface TodoWithUser extends Todo {
+  user: User | null,
+}
```

Comment:
Sergii Nosachenko told that it's bad practice to make types in react-acpp-env
It's better to make it in separate file it `types` folder